src/main/java/org/sonar/plugins/scm/jazzrtc/JazzRtcBlameCommand.java

```diff
@@ -65,11 +69,18 @@ private void blame(FileSystem fs, InputFile inputFile, BlameOutput output) {
 
     int exitCode = execute(cl, consumer, stderr);
     if (exitCode != 0) {
-      throw new IllegalStateException("The jazz annotate command [" + cl.toString() + "] failed: " + stderr.getOutput());
+      if(ArrayUtils.contains(UNTRACKED_BLAME_RETURN_CODES, exitCode)) {
+        //skips file silently
+        return;
+      }
+      else {
```

Comment:
![New](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/new.png) ![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "else" on the same line that the previous closing curly brace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)